### PR TITLE
Vitamins, Protein, new reagents for almost every food, new recipes

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Many foods now have vitamin or protein in addition to nutriment. They behave like they do in /tg/. I also added recipes for the plants I've added recently.

**Changelog**

:cl: Rane
- add: Most healthy foods now contain some vitamins and proteins, which have a chance to heal you slightly.
- add: There are new recipes using nettles, fly amanita, or plasma sheets.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 1]
```diff
@@ -104,6 +104,15 @@
...
     slice: FoodCakeCarrotSlice
+  - type: SolutionContainerManager #TODO Oculine
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 20
+        - ReagentId: Vitamin
+          Quantity: 5
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:115]: why can this not just be inherited

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 2]
```diff
@@ -116,6 +125,15 @@
...
     - state: carrot-slice
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 8
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 4
+        - ReagentId: Vitamin
+          Quantity: 1
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:136]: same with these

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml [hunk 3]
```diff
@@ -264,6 +313,19 @@
...
+        maxVol: 85
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 30
+        - ReagentId: Vitamin
+          Quantity: 5
+        - ReagentId: Iron
+          Quantity: 10
+        - ReagentId: Omnizine
+          Quantity: 30
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:328]: thats a lot of omnizine

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml (hunk 3 above)
Comment (Elijahrane) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:328]: Not cookable right now + there's other reasons you wouldn't want to eat it with its special mechanics

Comment (Elijahrane) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:328]: I added stuff for a lot of food that isn't actually able to be made right now but will be in the future

Comment (mirrorcult) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:328]: sure

**Pull request merged**